src/table/virtualized-table/hooks/use-dynamic-row-height.ts

```diff
@@ -170,7 +170,7 @@ const useDynamicRowHeight = ({
       const row = rows[rowIdx] ?? {};
       Object.values(row).forEach((cell) => {
         if (typeof cell === 'object') {
-          setCellSize(cell.qText ?? '', rowIdx, cell.pageColIdx);
+          setCellSize(cell.qText ?? '', rowIdx, cell.pageColIdx, cell.isNumeric);
```

Comment:
just a question our of curiosity: 
why using `mutableSetCellSize` in line 132 (inside of loop)
but in here we are **inside a loop again** but using `setCellSize`?

is this function `resizeVisibleCells` only being called once at when user is done with resizing? (if so, i think that should be fine to use `setCellSize` here then)

Reply:
`mutableSetCellSize` did not exist when this function was created and as it was not really needed. There might be some performance optimizations that in using `mutableSetCellSize` instead. But that is not for this PR.